metat_tasks.wdl

```diff
@@ -0,0 +1,665 @@
+         File final_filtered = "~{qcdir}/~{prefix}_filtered.fastq.gz"
+         File final_filtered_stats= "~{qcdir}/~{prefix}_filterStats.txt"
+         File final_filtered_stats2 = "~{qcdir}/~{prefix}_filterStats2.txt"
+         File final_rqc_info = "~{qcdir}/~{prefix}_rqc.info"
+         File final_rqc_stats = "~{qcdir}/~{prefix}_qc_stats.json"
+
+         # metaT_Assembly
+         File final_tar_bam = "~{assemdir}/~{prefix}_bamfiles.tar"
+         File final_contigs = "~{assemdir}/~{prefix}_contigs.fna"
+         File final_scaffolds = "~{assemdir}/~{prefix}_scaffolds.fna"
+         File final_readlen = "~{assemdir}/~{prefix}_readlen.txt"
+         File final_sam = "~{assemdir}/~{prefix}_pairedMapped.sam.gz"
+         File final_bam = "~{assemdir}/~{prefix}_pairedMapped.bam.gz"
+         File final_asmstats = "~{assemdir}/~{prefix}_scaffold_stats.json"
+         File final_asm_info = "~{assemdir}/~{prefix}_assy.info"
+         File final_asm_log = "~{assemdir}/~{prefix}_spades.log"
+
+         # mg_annotation
+         File final_proteins_faa = "~{annodir}/~{prefix}_proteins.faa"
+         File final_structural_gff = "~{annodir}/~{prefix}_structural_annotation.gff"
+         File final_ko_ec_gff = "~{annodir}/~{prefix}_ko_ec.gff"
+         File final_gene_phylogeny_tsv = "~{annodir}/~{prefix}_gene_phylogeny.tsv"
+         File final_functional_gff = "~{annodir}/~{prefix}_functional_annotation.gff"
+         File final_ko_tsv = "~{annodir}/~{prefix}_ko.tsv"
+         File final_ec_tsv = "~{annodir}/~{prefix}_ec.tsv"
+         File final_lineage_tsv = "~{annodir}/~{prefix}_scaffold_lineage.tsv"
+         File final_stats_tsv = "~{annodir}/~{prefix}_anno_stats.tsv"
+         File final_cog_gff = "~{annodir}/~{prefix}_cog.gff"
+         File final_pfam_gff = "~{annodir}/~{prefix}_pfam.gff"
+         File final_tigrfam_gff = "~{annodir}/~{prefix}_tigrfam.gff"
+         File final_smart_gff = "~{annodir}/~{prefix}_smart.gff"
+         File final_supfam_gff = "~{annodir}/~{prefix}_supfam.gff"
+         File final_cath_funfam_gff = "~{annodir}/~{prefix}_cath_funfam.gff"
+         File final_crt_gff = "~{annodir}/~{prefix}_crt.gff"
+         File final_genemark_gff = "~{annodir}/~{prefix}_genemark.gff"
+         File final_prodigal_gff = "~{annodir}/~{prefix}_prodigal.gff"
+         File final_trna_gff = "~{annodir}/~{prefix}_trna.gff"
+         File final_rfam_gff = "~{annodir}/~{prefix}_rfam.gff"
+         File final_product_names_tsv = "~{annodir}/~{prefix}_product_names.tsv"
+         File final_crt_crisprs = "~{annodir}/~{prefix}_crt.crisprs"
+         File final_imgap_version = "~{annodir}/~{prefix}_anno.info"
+         File final_renamed_fasta = "~{annodir}/~{prefix}_renamed_contigs.fna"
+         File final_map_file = "~{annodir}/~{prefix}_map.fna"
+
+         # metaT_ReadCounts
+         File final_count_table = "~{readmap}/~{prefix}.readcount"
+         File? final_count_ig = "~{readmap}/~{prefix}.readcount.intergenic"
```

Comment:
suggest naming this with the same suffix as JGI does `.rnaseq_gea.intergenic.txt`